Documentation/HowToUseETW.md

```diff
@@ -0,0 +1,46 @@
+# Supported
+ETW logs are available on CoreCLR only currently.
+
+
+# How to Collect ETW logs use logman
+
+
+## Start the logging
+
+ * Option 1: Generate a single file that does circular logging with Verbose level, For example
+ 
+ ```
+logman start WCFETWTracing -p "Microsoft-Windows-Application Server-Applications" 0xFFFFFFFF  0x5 -bs 64 -nb 120 320 -ets -ct perf -f bincirc -max 500 -o c:\temp\WCFEtwTrace.etl
```

Comment:
Do you think it's necessary here to talk about what each parameter here does, or perhaps add a link to MSDN that states what they do?